script.js

```diff
@@ -0,0 +1,123 @@
+import { Carrito } from './Carrito.js';
+import { Producto } from './Producto.js';
+
+// Crear una instancia del carrito
+let carrito = new Carrito();
+
+// Definición de productos disponibles
+let productos = [
+    { nombre: "Sweater Rainbow", precio: 15000 },
+    { nombre: "Sweater Molly", precio: 17000 },
+    { nombre: "Square Orange", precio: 20000 },
+    { nombre: "Square Blue", precio: 20000 },
+    { nombre: "Square Pink", precio: 20000 },
+    { nombre: "Square Red", precio: 20000 }
+];
+
+// Elementos del DOM
+const carritoSection = document.getElementById('carrito');
+const vaciarCarrito = document.getElementById('vaciarCarrito');
+const finalizarCompra = document.getElementById('finalizarCompra');
+const mensajeNotificacion = document.getElementById('mensajeNotificacion');
+
+// Función para agregar productos al carrito
+document.querySelectorAll('.agregar-carrito').forEach(button => {
+    button.addEventListener('click', function() {
+        const productName = this.parentElement.querySelector('p').textContent;
```

Comment:
Usa [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) para acceder de manera consistente a los atributos data-name y data-price.
```suggestion
const productName = this.dataset.name;
const productPrice = parseFloat(this.dataset.price);```